src/index.html

```diff
@@ -8,6 +8,73 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
+
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars stars--1">
```

Comment:
add extra line between _multiline_ siblings 
```suggestion
      </div>
      
      <div class="stars stars--1">
```